custom_components/kumo/kumo_cloud_setup.py

```diff
@@ -1,38 +1,44 @@
 #!/usr/bin/env python3
-""" Fetch required info from KumoCloud and produce HomeAssistant configuration
```

Comment:
There's no need to touch this file at all. It is not used in Home Assistant.

Reply:
I wasn't aware that this would not be used in homeassistant so I still ran it through the tests which complained about the whitespace between """ and Fetch..., also it required a period at the end of these comments...

Reply:
fixed here https://github.com/dlarrick/hass-kumo/pull/19/commits/278bcffad790ef53503777ce7f16ad358d9a2989